---
title: Deep generative models for single-cell transcriptomics
summary: We develop a collection of DGMs for principled, scalable and fully-probabilistic analysis of multiple scRNA-seq datasets. Examples include scVI, scANVI, gimVI and totalVI. 
tags:
- Deep Generative models
- Variational Inference
- scRNA-seq
date: "2019-10-07T00:00:00Z"

# Optional external URL for project (replaces project detail page).
external_link: ""

image:
  caption: An artistic representation of single-cell RNA sequencing.
  focal_point: Smart

links:
- icon: github
  icon_pack: fab
  name: Contribute
  link: https://github.com/YosefLab/scVI
url_code: ""
url_pdf: ""
url_slides: ""
url_video: ""

# Slides (optional).
#   Associate this project with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides = "example-slides"` references `content/slides/example-slides.md`.
#   Otherwise, set `slides = ""`.
slides: 
---

Single-cell Variational Inference (scVI) is a fully-probabilistic model for single-cell RNA-sequencing data. scVI is a deep generative model whose inference uses variational methods and is amortized with neural networks. Consequently, scVI is fast and fits one million cells in under a few hours. Furthermore, scVI is flexible and achieves high accuracy for dimensionality reduction, imputation and differential expression. In addition to the [journal publication](/publication/sc-vi/), we recommend the following ressources to get more familiar with scVI:

* the [podcast](https://bioinformatics.chat/scvi) recorded on the bioinformatics chat
* the [YosefLab blog](https://yoseflab.github.io/) usually includes discussions about special topics
* our [codebase](https://github.com/YosefLab/scVI), which includes user-friendly notebooks 
* the [News & View](https://www.nature.com/articles/s41592-018-0230-9) in Nature Methods
* our [post](https://bair.berkeley.edu/blog/2018/12/05/genes/) on the Berkeley AI Research blog
